spec/backend_client_spec.lua

```diff
@@ -2,14 +2,18 @@ local _M = require('apicast.backend_client')
 local configuration = require('apicast.configuration')
 local test_backend_client = require 'resty.http_ng.backend.test'
 local ReportsBatch = require 'apicast.policy.3scale_batcher.reports_batch'
+local backend_calls_metrics = require 'apicast.metrics.3scale_backend_calls'
 
 describe('backend client', function()
 
   local test_backend
   local options_header_oauth = 'rejection_reason_header=1'
   local options_header_no_oauth = 'rejection_reason_header=1&no_body=1'
 
-  before_each(function() test_backend = test_backend_client.new() end)
+  before_each(function()
+    test_backend = test_backend_client.new()
```

Comment:
This should be not necessary with the changes from https://github.com/3scale/apicast/pull/904/commits/576fa67570358ffdeba4381b5693464b3882b33c